src/test/java/cz/cvut/kbss/termit/persistence/dao/VocabularyDaoTest.java

```diff
@@ -180,7 +180,7 @@ void updateGlossaryMergesGlossaryIntoCorrectRepositoryContext() {
         });
 
         final Glossary result = em.find(Glossary.class, vocabulary.getGlossary().getUri(),
-                descriptorFactory.glossaryDescriptor(vocabulary));
+            descriptorFactory.glossaryDescriptor(vocabulary));
```

Comment:
Again, many style changes - I thus expect that the previous version was not formatted properly w.r.t .editorconfig.

Reply:
Hopefully fixed with updated .editorconfig.